src/api/http.rs

```diff
@@ -160,6 +162,33 @@ fn status(bob: State<BobServer>) -> Json<Node> {
     Json(node)
 }
 
+#[get("/nodes")]
+fn nodes(bob: State<BobServer>) -> Json<Vec<Node>> {
+    let mapper = bob.grinder().backend().mapper();
+    let mut nodes = vec![];
+    let vdisks = collect_disks_info(&bob);
+    for node in mapper.nodes().values() {
+        let vdisks: Vec<VDisk> = vdisks
+            .iter()
+            .cloned()
```

Comment:
may be it's better to use filter_map and then clone filtered vdisks to reduce clones.